tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -566,10 +627,14 @@ impl<'a, N> fmt::Display for FullCtx<'a, N> {
 }
 
 #[cfg(not(feature = "ansi"))]
-impl<'a, N> fmt::Display for FullCtx<'a, N> {
+impl<'a, N> fmt::Display for FullCtx<'a, N>
+where
+    S: Subscriber + for<'lookup> LookupSpan<'lookup> + LookupMetadata,
+    N: for<'writer> FormatFields<'writer> + 'static,
+{
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let mut seen = false;
-        self.ctx.visit_spans(|_, span| {
+        self.ctx.visit_spans(|span| {
```

Comment:
i dont thinkl this will compile currently, since you changed `visit_spans` to be over IDs instead of `SpanRef`s...can we change it back?

Reply:
I switched `visit_spans` to operate over `SpanRef`s.